concent_api/core/tests/test_unit_model.py

```diff
@@ -1,16 +1,25 @@
 import pytest
 from django.core.exceptions import ValidationError
+from django.test import override_settings
+from django.core.exceptions import ValidationError
 
 from golem_messages import factories
 from golem_messages import message
+from golem_messages.factories import tasks
+from golem_messages.factories.tasks import ComputeTaskDefFactory
+from golem_messages.factories.tasks import WantToComputeTaskFactory
```

Comment:
Please change imports. You imported modules that are already imported. You should use `from golem_messages import factories` and use all factories from this import.